DataGeneration/dataGen.js

```diff
@@ -1,88 +1,11 @@
-const faker = require('faker');
+const itemWrite = require('./itemsGen.js');
+const reviewWrite = require('./reviewsGen.js');
 
-const createCsvStringifier = require('csv-writer').createObjectCsvStringifier;
-const csvStringifier = createCsvStringifier({
-    header: [
-        {id: 'id', title: 'ID'},
-        {id: 'title', title: 'TITLE'},
-        {id: 'description', title: 'DESC'},
-        {id: 'originalPrice', title: 'OGPRICE'},
-        {id: 'salePrice', title: 'SALEPRICE'},
-        {id: 'colors', title: 'COLORS'},
-        {id: 'sizes', title: 'SIZES'},
-        {id: 'liked', title: 'LIKED'},
-        {id: 'inStock', title: 'INSTOCK'},
-        {id: 'reviews', title: 'REVIEWS'},
-    ]
-});
-var fs = require('fs');
+const genData = async () => {

```

Comment:
Digging this, I like that you're going back to optimize your code so we can all have an easier time following it. Funny how we use async to make code synchronized. lol